libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -392,9 +392,9 @@ export const mapEstateInfo = (syslaData: DanarbuUppl): EstateInfo => {
       ? syslaData.eignir
           .filter(
             (a) =>
+              a?.tegundAngalgs !== undefined &&
```

Comment:
FYI: Wanted to reverse the boolean checking order but for some reason Typescript complains if we don't explicitly ask for !== undefined. It's serving the same purpose here.